.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -24,6 +24,8 @@ Please check existing issues to see if your bug has already been reported before
 Please provide a code snippet or steps to consistently reproduce the experienced behavior.
 [DO NOT INCLUDE ANY PERSONAL OR SENSITIVE DATA IN YOUR SUBMITTED CODE]
 
+**Attach the logs (if this is an error when making an API call)**
```

Comment:
Add a blank line after these bold sections (here and elsewhere) if you want the part in bold on a separate line. See the preview here:

https://github.com/googleads/google-ads-dotnet/blob/baf7831644b541ead89c82dd12302adc9a5041cb/.github/ISSUE_TEMPLATE/bug_report.md